py/picca/delta_extraction/data_catalogues/desi_healpix.py

```diff
@@ -62,6 +69,10 @@ def __init__(self, config):
         """
         self.logger = logging.getLogger(__name__)
 
+        self.use_non_coadded_spectra = config.getboolean("use non-coadded spectra")
```

Comment:
Usually, I add this in a `_parse_config` method (as in DesiTile, for example), it might be good to add it here too in case we add more options in the future

Reply:
note that this addition was less straight-forward than expected, as when I first had the method included, the `_parse_config` of the parent class was silently ignored without tests failing (but failing at runtime due to eg missing input_directory)